### ISSUE TITLE
[Accessibility][ScreenReader]: Going back to text elements with shift-tab does not read correctly in a screen reader.

### ISSUE THREAD

**Issue by rp-dev** [microsoft/BotFramework-WebChat#3117]
## Screenshots

![image](https://user-images.githubusercontent.com/50723361/79809121-9e881080-8324-11ea-8ed9-b98cd988592d.png)

## Version

v4

## Describe the bug

*Arrive at *I Accept Button with JAWS ON and Keyboard Tab. Then press Shift + Tab to arrive at previous links such as "Privacy Statement", "Terms and Conditions"  

Actual:  The focus comes to these links but JAWS announces "I Accept" button every time. 

Expected:  Based on which link has JAWS focus, it should announce the link name and role correctly. 

## Additional context

Environment:  Windows 10 + Chrome 

[Bug]

**Comment by compulim** [comment]
This is fixed in #3287.
